docs/get-started/get-started.md

```diff
@@ -159,10 +159,7 @@ In your layout xml file, you need to add the following view:
     <androidx.compose.ui.platform.ComposeView
         android:id="@+id/gravatarComposeView"
         android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintEnd_toEndOf="parent" />
```

Comment:
These are `ConstraintLayout` specific but the `ComposeView` can be a child of other parent layouts so no need to add those.